### PR TITLE
feat: add scroll testnet

### PR DESCRIPTION
**Pull request by milapsheth**
[AXE-1975]

[AXE-1975]: https://axelarnetwork.atlassian.net/browse/AXE-1975?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/data/cosmos_chains.json [hunk 1]
```diff
@@ -224,6 +218,12 @@
       "network_id": "fetch",
       "image": "/images/chains/fetch.svg"
     },
+    {
+      "id": "haqq",
+      "name": "Haqq",
+      "network_id": "haqq",
+      "image": "/images/chains/haqq.svg"
```
Comment (ilyalukyanov) [inline src/data/cosmos_chains.json:225]: Image is missing:
image

File: src/data/evm_assets.json
Comment (ilyalukyanov) [inline src/data/evm_assets.json:1]: This was auto-generated, right? No point in reviewing?

Comment (milapsheth) [reply src/data/evm_assets.json:1]: yeah, auto generated files

**Review by ilyalukyanov — APPROVED** [review]

File: src/data/cosmos_chains.json (hunk 1 above)
Comment (milapsheth) [reply src/data/cosmos_chains.json:225]: @kalidax

**Pull request merged**